hicommonwealth/commonwealth
# Fix account redirect

## Link to Issue
Closes: #6292

## Description of Changes
- Updates the profile redirect component so it properly redirects `/account/:address` to `profile/id/:profile_id`
- Should resolve these errors: https://app.rollbar.com/a/commonwealth/fix/item/Commonwealth/6176#detail
- Removes an ancient route from `setupAppRoutes.ts`

## Test Plan
- Follow `Custom-Domains.md` instructions to locally set up `forums.dydx.community` as a custom domain
- Start the proxy and server
- Visit https://forums.dydx.community/account/0x2cE1F5d4f84B583Ab320cAc0948AddE52a131FBE
  - You should be redirected to a valid profile at https://forums.dydx.community/profile/id/118532

## Deployment Plan

1. 

## Other Considerations

-
